man/ctags-lang-verilog.7.rst.in

```diff
@@ -162,6 +162,28 @@ because they cannot be overridden.
 	P3	input.sv	/^	parameter  P3 = "parameter";$/;"	c	module:no_parameter_port_list	parameter:
 	L7	input.sv	/^	localparam L7 = "localparam";$/;"	c	module:no_parameter_port_list
 
+Supported Roles
+~~~~~~~~~~~~~~~
+
+.. code-block:: console
+
+	$ ./ctags --list-roles=SystemVerilog
+	#KIND(L/N) NAME ENABLED DESCRIPTION
+	m/module   decl on      declaring instances
+
+	$ ./ctags --list-roles=Verilog
+	#KIND(L/N) NAME ENABLED DESCRIPTION
+	m/module   decl on      declaring instances
+
+The parsers extract modules names used in instance declarations as
+reference tags. ``decl`` is the role for the tags. See "TAG ENTRIES"
```

Comment:
How about:

> The parser extracts module-names (or names of modules) used in instance declarations as reference tags.

We can say "The parsers extraces ...".  If we choose this, we have to change other sentences in this manual.

> The man page about SystemVerilog/Verilog parser*s* for Universal Ctags

> This man page describes about the SystemVerilog/Verilog parser*s* for Universal Ctags.

I prefer singulars.

Reply:
Updated.